src-docs/src/views/notification_event/notification_event_example.js

```diff
@@ -0,0 +1,218 @@
+import React, { Fragment } from 'react';
+import { renderToHtml } from '../../services';
+import { GuideSectionTypes } from '../../components';
+import { Link } from 'react-router-dom';
+import { EuiNotificationEventMeta } from '../../../../src/components/notification/notification_event_meta';
+import {
+  EuiNotificationEvent,
+  EuiText,
+  EuiContextMenuItem,
+  EuiSpacer,
+  EuiCode,
+  EuiTextColor,
+} from '../../../../src/components';
+import { notificationEventReadButtonConfig } from './playground';
+import { EuiNotificationEventPrimaryAction } from './props';
+
+import NotificationEvent from './notification_event';
+const notificationEventSource = require('!!raw-loader!./notification_event');
+const notificationEventHtml = renderToHtml(NotificationEvent);
+
+import NotificationsFeed from './notifications_feed';
+const notificationsFeedSource = require('!!raw-loader!./notifications_feed');
+const notificationsFeedHtml = renderToHtml(NotificationsFeed);
+
+const notificationEventSnippet = `<EuiNotificationEvent
+  id={id}
+  meta={{
+    type: 'Alert',
+    iconType: 'logoMaps',
+    eventName: 'alert-maps-01',
```

Comment:
This prop, it seems, will be much less often used than `iconAriaLabel`. Maybe let's swap them?
```suggestion
    iconAriaLabel: 'Maps',
```